clic/include/tier7.hpp

```diff
@@ -210,7 +210,7 @@ closing_labels_func(const Device::Pointer & device, const Array::Pointer & src,
 
 /**
  * @name erode_connected_labels
- * @category 'label processing', 'in assistant', 'bia-bob-suggestion'
+ * @category 'label processing', 'in assistant'
```

Comment:
Btw @StRigaud here it says "category" and in all other places "note"
```suggestion
 * @note 'label processing', 'in assistant'
```

Reply:
a mistake, can be switch back to note